contracts/Lens/VenusLens.sol

```diff
@@ -19,6 +20,18 @@ interface LensInterface {
 }
 
 contract VenusLens is ExponentialNoError {
+
+    using SafeMath for uint;
+
+    /// @notice Blocks Per Day
+    uint public constant BLOCKS_PER_DAY = 28800;
+
+    /// @notice comptrollerAddress
+    address public comptrollerAddress;
+
+    constructor(address _comptrollerAddress) public {
+        comptrollerAddress = _comptrollerAddress;
+    }
```

Comment:
I propose we keep the references to the protocol as method parameters. This way, when we have isolated pools, we can just pass different `comptroller` or `markets` arguments without having to redeploy VenusLens